src/SmartComponents/PatchSet/PatchSetAssets.js

```diff
@@ -37,7 +37,7 @@ export const patchSetRowActions = (openPatchSetEditModal, handlePatchSetDelete)
         }
```

Comment:
And 'Edit patch set' is staying in row actions

Reply:
@gabipodolnikova thank you for the quick look. Nope, I will rename this line as well :)

Reply:
@gabipodolnikova I have fixed ^. You can find it easly on a new commit.